configuration_manager/config_logger.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Forschungszentrum Jülich GmbH and Aix-Marseille Université
+# "Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements; and to You under the Apache License,
+#  Version 2.0."
+
+import os
+import logging
+import logging.config
+from functools import reduce
+import operator
+
+
+class ConfigLogger(object):
```

Comment:
I have the feeling that the functions in this class need some comments. 
It is not clear in one viewing for all functions what is expected behavior.